internal/tools/go.mod

```diff
@@ -6,63 +6,272 @@ require (
 	github.com/dmarkham/enumer v1.5.5
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/mock v1.6.0
+	github.com/hexdigest/gowrap v1.2.5
 	github.com/mattn/goveralls v0.0.11
 	github.com/mgechev/revive v1.2.1
 	github.com/uber/tcheck v1.1.0
-	github.com/vektra/mockery/v2 v2.14.0
+	github.com/vektra/mockery/v2 v2.0.3
```

Comment:
This was modified in https://github.com/uber/cadence/pull/4895 and actually v2.14.0 requires go version 1.18 which we don't support yet. So I'm downgrading it here.

Reply:
thank you!  I was using 1.18, didn't notice.